app/lib/track/src/track.cpp

```diff
@@ -7,6 +7,9 @@
 const size_t Track::playBufferSize = 65536;    // 64k
 const size_t Track::recordBufferSize = 131072; // 128k
 
+// For reverse track ---- NEEDED?
+#define BUFFER_SIZE 1024
```

Comment:
Like I mentioned earlier, this doesn't need to be a `#define` macro. Also, the name `BUFFER_SIZE` is a bit vague, especially since this class has `playBufferSize` and `recordBufferSize` variables.